Claire51 onboarding

claire-lee-51 opened the pull request:
Completed both basic firmware and hardware projects; just misnamed firmware project when it was initially pushed (listed it as "project 2"). 

Not sure if basic hardware project works.

claire-lee-51 committed:
proj 1 (prog 1) completed

claire-lee-51 committed:
project 2 - completed but most likely not working

claire-lee-51 committed:
Completed Project 2 (Firmware); just pushed project wrong at first.

claire-lee-51 committed:
completed basic firmware project, just pushed wrong initially.

anav512 commented on Firmware/project_1/src/pins.cpp:
Minor typo here. You put the semicolon inside the bracket.

anav512 reviewed (commented):
unable to open your kiCAD files

claire-lee-51 committed:
Fixed error of misplaced semi-colon on pins.cpp, made best attempt at
Firmware Challenge Project. Did best to clean up Firmware Introduction
Project 1.

claire-lee-51 commented:
Fixed mistake on pins.cpp, where a semi-colon was misplaced within the defined variable.
Tried fixing Firmware Project 1, since I realized that I did it wrong.
Attempted(?) Challenge Firmware Project 2, but no idea if I was able to successfully do it the way that it was meant to. Did try my best.
For kiCAD files, I worked on them in KICAD 9.0, therefore making it incompatible. I got the easy hardware project checked over on 10/2/2025 by one of the club leaders, and was approved with no errors.

Would just like/need feedback on firmware project 1 and 2 :).
